packages/pieces/discourse/src/lib/actions/new-pm-to-user.action.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { HttpMethod, httpClient } from '@activepieces/pieces-common';
+import { discourseAuth } from '../../index';
+import { Property, createAction } from "@activepieces/pieces-framework";
+
+
+
+export const newPrivateMessage = createAction({
+    auth: discourseAuth,
+    name: 'new_private_message',
+    description: 'Create a new private message in Discourse',
+    displayName: 'New Private Message',
```

Comment:
Can you please change to "Send Private Message"

Reply:
done